test/mormot2tests.lpi

```diff
@@ -1,23 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please don't publish your own custom .lpi file

Reply:
Reverted .lpi file to master branch. Maybe we should add this file to git-ignore-file?